litert/vendors/intel_openvino/dispatch/device_context.cc

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "litert/vendors/intel_openvino/dispatch/device_context.h"
+#if __ANDROID__
```

Comment:
I suppose you are guarding this header because you want to be able target platforms other than Android, like linux?  If so, don't we also need to guard other places in the .cc file that use Android-only symbols, e.g. like in function `GetFdFromUnixHandle`?

Feel free to address this in the current pull request, or leave support for other platforms to a later pull request.

Reply:
Done